src/components/TopBar/CallButton.vue

```diff
@@ -191,6 +192,16 @@ export default {
 	}
 }
 
+.success {
+	color: white;
+	background-color: #35CC00;
```

Comment:
Can we use the variables.scss for this? So in case they are reused its the same, i'd like to not use any non variable colors